content/300-guides/125-development-environment/050-environment-variables/index.mdx

```diff
@@ -25,7 +25,7 @@ Taking the `TEMP` environment variable as an example, one can query its value to
 
 Prisma always reads environment variables from the system's environment.
 
-When you install Prisma for the first time it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are put into the system's environment, where Prisma can read it and use it.
+When you initially create your Prisma setup with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use Prisma CLI or Prisma Client, the `.env` file content and the variables defined in there are put into the system's environment, where Prisma can read it and use it.
```

Comment:
```suggestion
When you initially create your Prisma setup with `prisma init`, it creates a convenience `.env` file for you to set your [`connection url`](/reference/database-reference/connection-urls) as an environment variable. When you use the Prisma CLI or Prisma Client, the `.env` file and the variables defined in it are put into the system's environment, where Prisma can read it and use it.
```

Reply:
(outdated now, have to create again so can be accepted)